components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/par/OAuth2ParEndpoint.java

```diff
@@ -115,16 +123,18 @@ private Response handleParClientException(ParClientException exception) {
         } else {
             responseBuilder = Response.status(HttpServletResponse.SC_BAD_REQUEST);
         }
+        log.error("PAR client Exception: ", exception);
```

Comment:
This should be a debug log. And instead of saying "PAR client Exception: "let's add a proper text based on the error  context. Or have a generic wording like "Client error while handling the request"